scripts/python/testbedApi.py

```diff
@@ -0,0 +1,329 @@
+# Python testing utility classes and functions
```

Comment:
I don't know python and will be unable to maintain this code. I'd prefer to keep the go code and not introduce new tools if not necessary.